.circleci/config.yml

```diff
@@ -7,5 +7,14 @@ jobs:
     steps:
         - checkout
         - run:
-            name: Pass
-            command: echo 'true'
+            name: Install core utilities
+            command: make install
+        - run:
+            name: Setup the Virtual Environment
+            command: make venv && source venv/bin/activate
```

Comment:
Not sure if sourcing/pipenv will work the way you expect in CodeBuild. If so, great! If not, you can always run the binaries directly, for example `venv/bin/pylint`

Reply:
Well I'm not sure if it works as I expects, but its passing! I"ll leave it as is for now until we have a reason to change.